docs/source/models/gbd2017/risk_attributable_causes/iron_deficiency/index.rst

```diff
@@ -4,5 +4,152 @@
+populations.
+
+**Dietary iron deficiency anemia** is a specific type of iron deficiency anemia 
+that is due to inadequate dietary intake of iron, leading to inadequate iron 
+levels in the body and a subsequent deficiency of hemoglobin in the blood.
+
+Modeling Iron Deficiency in GBD 2017
+------------------------------------
+
+In GBD 2017, the cause dietary iron deficiency is a population attributable 
+fraction (PAF) of 1 cause with the iron deficiency risk factor. This means 
+that 100% of the dietary iron deficiency cases are attributable to the iron 
+deficiency risk factor. Notably, there are additional causes other than 
+dietary iron deficiency that are attributable to the iron deficiency risk 
+factors (ex: maternal disorders).
+
+Additionally, there is an anemia *impairment* modeled in GBD 2017 that 
+represents **all** forms of anemia that are attributable to several causes, 
+including causes such as hemoglobinopathies and hemolytic anemias that are not 
+causally attributable to iron deficiency. 
+
+Iron Deficiency Risk Factor (REI ID = 95)
++++++++++++++++++++++++++++++++++++++++++
+
+The iron deficiency risk factor in GBD 2017 is modeled as a distribution of 
+hemoglobin concentrations.
+
+.. todo::
+
+	Describe the population and distribution
+
+Modeling Strategy for the Iron Deficiency Risk Factor
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
 .. todo::
-   Describe PAF-of-1 cause model
+
+	Describe anemia envelope and causal attribution
+
+Risk Factor Hierarchy
+^^^^^^^^^^^^^^^^^^^^^
+
+.. image:: iron_risk_hierarchy.svg
+
+Dietary Iron Deficiency Cause (Cause ID = 95)
++++++++++++++++++++++++++++++++++++++++++++++
+
+The dietary iron deficiency cause in GBD 2017 that is 100% attributable to the 
+iron deficiency risk factor. The dietary iron deficiency cause in GBD is a 
+YLD-only cause, meaning that it contributes to morbidity, but not mortality.
+
+Modeling Strategy for the Dietary Iron Deficiency Cause
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The dietary iron deficiency cause in GBD 2017 is not modeled directly. Rather, 
+the dietary iron deficiency cause is estimated as the remaining cases of 
+anemia after all of the other anemic cases in the overall anemia envelope were 
+causally attributed to their respective attributable causes (ex: 
+hemoglobinopathies and hemolytic anemias).
+
+Cause Hierarchy
+^^^^^^^^^^^^^^^
+
+.. image:: iron_cause_hierarchy.svg
+
+Health States and Sequela
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The sequela associated with the dietary iron deficiency cause in GBD 2017 
+include mild iron deficiency anemia, moderate iron deficiency anemia, and 
+severe iron deficiency anemia. The severity of iron deficiency anemia is 
+determined by the WHO age- and sex- specific hemoglobin concentrations, as 
+described in the table below.
+
+.. list-table:: Hemoglobin Thresholds (g/L)
+   :widths: 15, 15, 15, 15
+   :header-rows: 1
+
```

Comment:
I am having trouble getting this table to render. If you spot an issue in my code please let me know!

Reply:
Is it possible that your 'commas used in :widths: ',' are creating the issue?

Reply:
In various list table examples online, it shows :widths: as never using commas, but I am not sure if that is mjust a trivial difference and the issue is something else but it's worth a try.

Reply:
The commas should be fine, according to the specification in https://docutils.sourceforge.io/docs/ref/rst/directives.html#list-table

@alibow , the problem is your indentation: The bullets are not lined up with the line above your list (i.e. ":header-rows: 1" - the asterisks should line up with the first colon in this line), so the compiler doesn't recognize the list as being part of the list-table directive.

Reply:
Looking at the whitespace characters in Atom, it looks like you have hard tab characters (\t) instead of spaces for your list, so the spacing might look right depending on your text editor settings, but it won't be recognized as correct `.rst`. You should delete the hard tabs in front of the list, and replace them with the correct number of spaces (3, it appears).

It looks like you have some hard tabs in other places as well (namely your todo boxes), which could potentially cause problems at some point, so it might be good to get rid of those as well.

Reply:
If you're using SublimeText, presumably there should be a setting that automatically inserts soft tabs (some number of spaces) instead of hard tabs when you hit the tab key, and shortcut keys to automatically increase or decrease the indentation level with the correct number of soft tabs.